### PR TITLE
QoL improvements for the prebuild UI

### PR DESCRIPTION
**Pull request by filiptronicek**
## Description

Adds the following to the prebuild detail page:
1. Persistence of the actively selected task via URL hash
2. A margin for tab triggers, so that the focus ring can fit in there properly
3. Disables analytics tracking on the buttons changing the currently displayed task, so that our analytics don't show us what users name their tasks.

## How to test

https://ft-prebuild-ui-qol.preview.gitpod-dev.com/workspaces

#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [x] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [x] /werft preemptible
      Saves cost. Untick this only if you're really sure you need a non-preemtible machine.
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
- [ ] with-monitoring
</details>

/hold

### CODE REVIEW

File: components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx [hunk 1]
```diff
@@ -63,7 +63,7 @@ export const PrebuildDetailPage: FC = () => {
     const { toast, dismissToast } = useToast();
     const [currentPrebuild, setCurrentPrebuild] = useState<Prebuild | undefined>();
     const [logNotFound, setLogNotFound] = useState(false);
-    const [selectedTaskId, setSelectedTaskId] = useState<string | undefined>(undefined);
+    const [selectedTaskId, actuallySetSelectedTaskId] = useState<string | undefined>(window.location.hash.slice(1));
```
Comment (filiptronicek) [inline components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx:66]: we could improve this with some basic input validation, it is very naive.

Comment (geropl) [reply components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx:66]: When opening the the prebuild details page directly (without any taskId passed) we should select the first task, and also focus that task's tab.
Atm I see those rendered, but none has focus, so also not streaming logs. :+1:

Comment (filiptronicek) [reply components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx]: Of course I forget to test the most important (default) case. Thanks for noticing, just push a fix 🤦

**Review by geropl — APPROVED** [review]
Code LGTM, tested and the changes do work. :heavy_check_mark: 

I see problems connecting to the log streams, but I think those will be resolved by your other PR, and are definitely unrelated to this one. :+1:

**Pull request merged**